ci/ray_ci/tests.env.Dockerfile

```diff
@@ -0,0 +1,10 @@
+# syntax=docker/dockerfile:1.3-labs
+# shellcheck disable=SC2148
+
+ARG BASE_IMAGE
+FROM "$BASE_IMAGE"
+
+ARG TEST_ENVIRONMENT_SCRIPT
+
+COPY "$TEST_ENVIRONMENT_SCRIPT" /tmp/post_build_script.sh
+RUN /tmp/post_build_script.sh
```

Comment:
this feels that you can just: `docker create`, `docker cp` the file in, and then `docker start`. feels a bit overkill to create a new container just for adding a script to run.

Reply:
you can also create a small docker volume, copy the script file into the volume (with a temp container), and then mount the volume into the container to run, if you just want to use `docker run` and rely on docker to rm the container after running.

Reply:
resolved offline. it is required to be shared with multiple parallel container runs.